### PR TITLE
Add python deps from homeassistant base image

### LINKED ISSUE #6: Psycopg2 missing
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

## Current Behavior
`ERROR (Recorder) [homeassistant.components.recorder] Error during connection setup: No module named 'psycopg2' (retrying in 3 seconds) `

Same as #5, the psycopg2 python library needs to be installed to support Postgres out of the box. It’s not clear to me how it’s getting installed in the official image without being in the requirements file.

### PR DESCRIPTION
**Pull request by thedanbob**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/docker-homeassistant/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

## Description:

Adds missing python dependencies from the Home Assistant base image.

## Benefits of this PR and context:

Closes #6 (and covers #5 as well).

## How Has This Been Tested?

Successfully running in place of the official homeassistant image on my server (fixing #6). I can't test the multiarch images directly but they build successfully.

### CODE REVIEW

**Comment by tobbenb** [comment]
Is there a special reason why you use the --only-binary flag? I would prefer to not use it, in case it needs to compile anything.

**Comment by thedanbob** [comment]
Just to match what the HA base image’s dockerfile does. I can remove it.

File: Dockerfile [hunk 1]
```diff
@@ -60,18 +61,32 @@ RUN \
...
+ mkdir -p \
+		/tmp/base && \
+  if [ -z ${HASS_BASE+x} ]; then \
+	HASS_BASE=$(curl -sX GET https://api.github.com/repos/home-assistant/docker/releases/latest \
+	| jq -r .tag_name); \
+ fi && \
+ curl -o \
+	/tmp/base.tar.gz -L \
+		"https://github.com/home-assistant/docker/archive/${HASS_BASE}.tar.gz" && \
+ tar xf \
```
Comment (Roxedus) [inline Dockerfile:73]: There is no need to grab the tarball here. Pip install can do links, ie `pip install -r https://raw.githubusercontent.com/home-assistant/docker/master/requirements.txt`

Comment (thedanbob) [reply Dockerfile:73]: I thought you might want to lock the base image version with HASS_BASE. If not, that's definitely the better option.

Comment (Roxedus) [reply Dockerfile:73]: We can still do that.
`pip install -r https://raw.githubusercontent.com/home-assistant/docker/${HASS_BASE}/requirements.txt`

Comment (thedanbob) [reply Dockerfile:73]: Actually, now that I think about it it's probably better to grab that version from the HA repo itself so it's always kept in sync: https://github.com/home-assistant/core/blob/dev/build.json

Comment (thedanbob) [reply Dockerfile:73]: Just saw your second message, excellent. Best of both worlds. I'll make those changes.

**Review by tobbenb — APPROVED** [review]

**Pull request merged**